src/_posts/platform/databases/2000-01-01-adminer.md

```diff
@@ -1,10 +1,22 @@
 ---
 title: Adminer
 nav: Adminer
-modified_at: 2022-04-28 00:00:00
+modified_at: 2023-12-15 12:00:00
 tags: databases addon adminer
 ---
 
+{% warning %}
+**The Adminer instance provided by Scalingo will be decommissioned and put
+offline on the 16th of February 2024 (2024-02-16).**\
```

Comment:
I think we can change the date and remove these tools on 1 April.
We should give about 3 months after the announcement.